public_dropin_gpu_environments/python311_genai/README.md

```diff
@@ -0,0 +1,39 @@
+# Python 3 GenAI Drop-In Template Environment
+
+This template environment can be used to create GenAI-powered custom models and includes common dependencies for workflows using OpenAI, Langchain, vector DBs, or transformers in PyTorch. 
+
+## Supported Libraries
+
+This environment is built for python 3 and has support for the following libraries.
+For specific version information and the complete list of included packages, see [requirements](requirements.txt).
+
+- openai
+- langchain
+- transformers
+- sentence-transformers
+- torch
+- faiss-cpu
```

Comment:
Not sure about this. Should we try to use `faiss-gpu` my understanding that there restriction about what cuda version it supports.